geode-core/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/DurableRegistrationDUnitTest.java

```diff
@@ -284,6 +278,14 @@ public void run2() throws CacheException {
     durableClientVM
         .invoke(() -> DurableRegistrationDUnitTest.registerKey(K4, Boolean.TRUE));
 
+    // Send clientReady message
+    durableClientVM.invoke(new CacheSerializableRunnable("Send clientReady") {
+      @Override
+      public void run2() throws CacheException {
+        CacheServerTestUtil.getCache().readyForEvents();
+      }
+    });
```

Comment:
While you're modifying this test, some warnings about deprecated methods and UUIDs can be avoided by changing this to:
```
    durableClientVM.invoke("Send clientReady",
        () -> CacheServerTestUtil.getCache().readyForEvents());
```

Reply:
Yeah, know. I resisted the urge to refactor these files as this is going into 1.15

Reply:
Done